setup.py

```diff
@@ -0,0 +1,33 @@
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
```

Comment:
you can use `packages=find_packages()` below (my preference, it's one less repetition of data that can go wrong), but if you aren't can you please cut this unused import?